go.mod

```diff
@@ -6,8 +6,7 @@ toolchain go1.22.2
 
 require (
 	github.com/container-storage-interface/spec v1.9.0
-	github.com/onsi/ginkgo/v2 v2.17.2
-	github.com/onsi/gomega v1.33.1
+	github.com/digitalocean/go-qemu v0.0.0-20230711162256-2e3d0186973e
 	github.com/openshift/custom-resource-status v1.1.2
```

Comment:
I'm trying to figure out why vendoring changed go.mod.  Do we need a separate patch to clean up go.mod before vendoring?

Reply:
It's good practice when using go modules to tidy. Tidying will make sure you have all the proper dependencies listed in your go.mod file. You can then run vendor to put all of the dependencies into the vendor folder (automated with `go mod vendor`)

Reply:
The usual sequence of commands is `go mod tidy`, then `go mod vendor`

Reply:
^ yes